branca/utilities.py

```diff
@@ -196,16 +199,14 @@ def split_six(series=None):
 
     """
     if pd is None:
-        raise ImportError('The Pandas package is required'
-                          ' for this functionality')
+        raise ImportError("The Pandas package is required" " for this functionality")
```

Comment:
Weird that Black doesn't concatenate the strings, but instead leaves the double quotes in the middle... But it's not harmful.